app/src/main/res/raw/login_form_detection_functions.js

```diff
@@ -0,0 +1,71 @@
+// Do not wrap this code into an anonymous function. Used in JavaScriptDetector.kt
```

Comment:
The comment here should impart the importance of wrapping in an anonymous function, rather than discouraging it. Maybe we can update it to say something along the lines of:

"// This code must be wrapped in an anonymous function which is done in JavaScriptDetector.kt to allow for dynamic changes before wrapping.